.github/styles/vocab.txt

```diff
@@ -0,0 +1,12 @@
+eg
```

Comment:
Why did you include eg?

Reply:
It was erroring out on the use of eg for exempli gratia, but we can remove it if we want to target and remove eg

Reply:
Yeah, I want "for example" everywhere.